server.js

```diff
@@ -0,0 +1,25 @@
+const express = require("express");
+const app = express();
+const cors = require("cors");
+const PORT = 4000;
+
+app.use(cors());
+app.use(express.json());
+
+const shopItemRouter = require("./routes/shopItem.js");
+app.use("/shopItem", shopItemRouter);
+
+const findIdRouter = require("./routes/findId.js");
+app.use("/shopItem/:id", findIdRouter);
+
+const tickStatusRouter = require("./routes/tickStatus.js");
```

Comment:
As discussed, we'll call this endpoint findByStatus instead. Maybe `shopItemByStatus` would be a better name